.github/workflows/pr_agent.yml

```diff
@@ -17,5 +17,8 @@ jobs:
         id: pragent
         uses: Codium-ai/pr-agent@main
         env:
-          OPENAI_KEY: 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
-          GITHUB_TOKEN: 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
+          OPENAI_KEY: ${{ secrets.OPENAI_KEY }}
```

Comment:
Good job on replacing hard-coded sensitive information with GitHub secrets. However, consider removing the commented-out lines with the hard-coded keys to avoid potential security risks. [important]